### DISCUSSION TITLE
请问文章的目录怎么生成？

### DISCUSSION THREAD

**Discussion by fujiwaranatsuho** [MaLuns/hexo-theme-async#107] [Q&A 疑难杂症]
我在主题的yml文件写了
toc:
  enable: true
  list_number: true
  post_title: true
  max_depth: 3
  min_depth: 1
但是文章依旧没有目录，是还需要写什么内容吗？

**Comment by MaLuns** [comment]
根据 md 文件里，标题生成的

**Comment by MaLuns** [comment]
> 我在主题的yml文件写了
> toc:
> enable: true
> list_number: true
> post_title: true
> max_depth: 3
> min_depth: 1
> 但是文章依旧没有目录，是还需要写什么内容吗？

配置开启了，会自动根据你 md 里标题生成，不需要其他配置

**Reply by fujiwaranatsuho** [reply]
我的文章里面h1（直接用的#+空格）的部分也没有变成目录

**Reply by MaLuns** [reply]
二级标题有吗，或者你打开你文章页面看看 把鼠标放文章标题上面 会出现目录按钮图标吗？没有话，那就是你上面配置没生效，看配置位置和格式对不对

**Reply by fujiwaranatsuho** [reply]
光标会变成手指，但是没有目录按钮。配置我是复制官方文档的内容，就直接写在主题yml文件的最末尾了，位置不对吗？

**Reply by fujiwaranatsuho** [reply]
我知道了！每篇文章前面有个toc:false。谢谢！
